justfile

```diff
@@ -39,7 +39,7 @@ test-superchain: clean-ops
 
 # Unit test all Go code in the validation module, and do not run validation checks themselves
 test-validation: clean-ops
-	TEST_DIRECTORY=./validation/... go run gotest.tools/gotestsum@latest --format testname -- -run '[^TestValidation|^TestPromotion|^TestGenesisAllocs]'
+	TEST_DIRECTORY=./validation/... SKIP_TESTMAIN=true go run gotest.tools/gotestsum@latest --format testname -- -tags "" -run '[^TestValidation|^TestPromotion|^TestGenesisAllocs]'
```

Comment:
Should we rename `SKIP_TESTMAIN` to something like `SKIP_GENESIS_ALLOCS`? I'm unsure about what the variable is for with this generic name. And does this replace the `^TestGenesisAllocs` filter?